Libplanet/Libplanet.csproj

```diff
@@ -61,7 +61,7 @@ https://docs.libplanet.io/</Description>
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Bencodex" Version="0.3.0-dev.a08e69324ddc8cf131e701b307eecba20bca1bf4" />
+    <PackageReference Include="Bencodex" Version="0.3.0-dev.1e890b6f026279f9ba485473c1579e784215880a" />
```

Comment:
As Bencodex has broken API/ABI-level compatibility from 0.3.0-dev.a08e69324ddc8cf131e701b307eecba20bca1bf4 to 0.3.0-dev.1e890b6f026279f9ba485473c1579e784215880a, I believe this change should go to the main branch instead of the 0.10-maintenance.